.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+      - Use this form to provide the following details:
+        - The bug that necessitates this cherry-pick.
+        - The PR(s) that fix the bug.
+        - The release to which the cherry-pick will be applied.
+        - The channel(s) to which the cherry-pick will be applied.
+- type: input
+  id: issue
+  attributes:
+    label: Issue
+    description: The bug that necessitates this cherry-pick. Remember to update the issue title with this info.
+    placeholder: "e.g. #11111"
+  validations:
+    required: true
+- type: input
+  id: pull_request
+  attributes:
+    label: Pull Request(s)
+    description: The PR(s) that fix the bug. Make sure they are merged and have passing [CI builds](https://app.circleci.com/pipelines/github/ampproject/amphtml?branch=main).
+    placeholder: "e.g. #22222, #33333"
+  validations:
+    required: true
+- type: input
+  id: release_tracker
+  attributes:
+    label: Release Tracker
+    description: The [tracker issue](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for the release to which the cherry-pick will be applied. Remember to update the issue title with this info.
+    placeholder: "e.g. #44444"
+  validations:
+    required: true
+- type: dropdown
+  id: channels
+  attributes:
+    label: Channels
+    description: The [release channels](https://github.com/ampproject/amphtml/blob/main/docs/release-schedule.md#release-channels) to which the cherry-pick will be applied. Remember to update the issue labels with this info.
+    multiple: true
+    options:
+      - Beta / Experimental
+      - Stable
+      - LTS
+  validations:
+    required: true
+- type: markdown
+  id: channel_instructions
+  attributes:
+    value: |
+      Select one or more channel(s) to which this cherry-pick is applicable.
+      - A cherry-pick to Stable will most likely require a cherry-pick to Beta / Experimental as well.
+      - Without this, it is possible that the fix could be lost when Beta goes to Stable.
+      - Once you select a set of channels, make sure the issue labels match your selection.
+- type: textarea
+  id: justification
+  attributes:
+    label: Justification
+    description: Why you believe this issue meets the [cherry-pick criteria](https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md#Cherry-picks).
+    placeholder: Provide a specific rationale for why this fix cannot wait until the next Stable release.
+  validations:
+    required: true
+- type: textarea
+  id: why_not_beta_experimental
+  attributes:
+    label: Why Not Beta / Experimental?
+    description: Why a Stable cherry-pick is necessary, but not Beta / Experimental.
+    placeholder: If the cherry-pick is going into Stable but not Beta / Experimental, provide a rationale (e.g. the PR is already part of the beta channel). Otherwise, skip this section.
```

Comment:
Is there an answer here besides "because it's not in Beta" that we expect or will find useful? Wondering if this question can be removed.